src/plugin.js

```diff
@@ -231,18 +261,33 @@ const onPlayerReady = (player) => {
  * to you; if not, remove the wait for "ready"!
  *
  * @function eme
+ * @param    {Object} [player]
+ *           A player object.
  * @param    {Object} [options={}]
  *           An object of options left to the plugin author to define.
  */
-const eme = function(options = {}) {
```

Comment:
This worked for me:

```
const eme = function(options = {}) {
  const player = this;

  this.ready(() => onPlayerReady(this));

  this.eme = {
    initializeMediaKeys(emeOptions = {}) {
      const mergedEmeOptions = videojs.mergeOptions(
        player.currentSource(),
        options,
        emeOptions
      );

      return initializeMediaKeys(player, mergedEmeOptions);
    },
    options
  };
};

// Register the plugin with video.js.
const registerPlugin = videojs.registerPlugin || videojs.plugin;

registerPlugin('eme', eme);
```

to be clear, I'm trying to avoid an anonymous function being the plugin factory here